### ISSUE TITLE
[hooks] Each item should have aria-selected="false" if not highlighted

### ISSUE THREAD

**Issue by silviuavram** [downshift-js/downshift#928]
**Problem description**:
For `useSelect` and `useCombobox`, `getItemProps` will not assign any `aria-selected` attribute to the item if it is not highlighted. It will only add `aria-selected="true"` to the highlighted item.

**Suggested solution**:
Add `aria-selected="false"` to all items that are not highlighted. This will ensure consistency with the `Downshift` component and will also follow ARIA specifications.
